### ISSUE TITLE
[NTR] telomere syndromes

### ISSUE THREAD

**Issue by ValWood** [monarch-initiative/mondo#1631]
I am wondering why there is not a grouping term for
telomere syndrome

definition (could be improved)
Short telomere syndromes (STSs) are accelerated aging syndromes often caused by inheritable gene mutations resulting in decreased telomere lengths.

(we used to have a grouping term in DO)

is_a premature aging syndrome MONDO_0019303

**Children terms 
If this term is added (it. used to be in DO), I can look into locating the child terms

**Comment by ValWood** [comment]
children
 diskeratosis congenita 
and
Uner Tan syndrome 
would be descendants

**Comment by ValWood** [comment]
Also, the telomerase RNA which we used to annotate directly to this term and is currently "disease-less"

**Comment by nicolevasilevsky** [comment]
what is Uner Tan syndrome? I don't see it in Mondo or OLS

**Comment by nicolevasilevsky** [comment]
@ValWood I added this term and sub-classified the children listed in Table 1 in this paper:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6035054/

**Comment by ValWood** [comment]
was one of the only things that did not map from DO
Uner Tan syndrome
https://github.com/monarch-initiative/mondo/issues/1571

**Comment by nicolevasilevsky** [comment]
@ValWood could you provide some evidence that Uner Tan syndrome is a telomere syndrome?

**Comment by ValWood** [comment]
Whoops that is a mistake. I had something one more  It's the disease  associated with
telomerase RNA (TERC),  which I only had mapped to "telomere syndrome"
 and Uner Tan, I mixed them up.

TERC causes
pulmonary fibrosis and/or bone marrow failure, Telomere-related, 2 

this one is a telomere syndrome

**Comment by nicolevasilevsky** [comment]
Should this term be a child of telomere syndrome (and its children)?
MONDO_0000148 'pulmonary fibrosis and/or bone marrow failure, telomere-related'

**Comment by ValWood** [comment]
Yes I think so according to OMIM.

**Comment by ValWood** [comment]
Hi, I  can';t find the telomere syndrome term, was this added or wasn't it appropriate?

**Comment by ValWood** [comment]
Some background.
We used to have a grouping terms for telomere syndromes.

This would include for example,
dyskeratosis congenita, autosomal dominant 1 (MONDO:0007485)
A dyskeratosis congenita that has_material_basis_in an autosomal dominant mutation of TERC on chromosome 3q26.2.

a definition
Short telomere syndromes (STSs) are accelerated aging syndromes often caused by inheritable gene mutations resulting in decreased telomere lengths. Consequently, organ systems with increased cell turnover, such as the skin, bone marrow, lungs, and gastrointestinal tract, are commonly affected.

**Comment by ValWood** [comment]
I can put this in a new ticket if you like...

So my issue is that at the moment,  trt1

dyskeratosis congenita, autosomal dominant 1 (MONDO:0007485)

do not get linked to premature ageing

**Comment by nicolevasilevsky** [comment]
Hi Val- looks like I make 'short telomere' syndrome an exact synonym of MONDO_0100137 telomere syndrome. Would you like 'short telomere syndrome' to be the primary label? Is 'telomere syndrome' an appropriate synonym?

**Comment by nicolevasilevsky** [comment]
> So my issue is that at the moment, trt1

> dyskeratosis congenita, autosomal dominant 1 (MONDO:0007485)

> do not get linked to premature ageing

Is there action required here?

**Comment by ValWood** [comment]
Yes, thats odd I could not find "telomere syndrome" when I searched MONDO. I can use that.
Does telomere syndrome have ancestry to "premature ageing syndrome"?

**Comment by ValWood** [comment]
dyskeratosis congenita, autosomal dominant 1 (MONDO:0007485)
should be is_a telomere syndrome

telomere_syndrome  should be  is_a "premature ageing syndrome"

I can open tickets for new/missing descendants of "telomere syndrome" as I find them.

**Comment by nicolevasilevsky** [comment]
'telomere syndrome' is a child of 'premature aging syndrome': https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0100137

**Comment by nicolevasilevsky** [comment]
> dyskeratosis congenita, autosomal dominant 1 (MONDO:0007485)
> should be is_a telomere syndrome

I added this, it will be available in the next release

**Comment by cmungall** [comment]
Reopening this, I think there are way too many children of STS now based on https://pubmed.ncbi.nlm.nih.gov/29804726/

This review includes things like some genetic forms of ILD, but I think this is wrong. ILD can be a *feature* of STS but not a *subtype*

E.g

<img width="676" alt="Image" src="https://github.com/user-attachments/assets/875f6d8d-7f04-41c6-9474-bef1e03a7342" />

See also 

- #4508

**Comment by ValWood** [comment]
facioscapulohumeral muscular dystrophy (MONDO:0001347) doesn't appear to be a telomere syndrome (it is caused by a defect in a splcing factor)

**Comment by sabrinatoro** [comment]
@nicolevasilevsky there is another issue about telomere syndromes. (tagging you while I look for the issue)
